src/phase-manager.ts

```diff
@@ -503,12 +723,10 @@ export class PhaseManager {
    * @param showText - (Optional, default `true`) Whether to show text
```

Comment:
```suggestion
   * @param showText - (Optional, default `true`) Whether to show text
   * @deprecated To be removed in a future PR
```
Also add to `queueStatStageChangePhase`.

Reply:
Actually there's literally only 2 callsites for `queueStatStageChangePhase` and 3 for `toLoginScreen`, you can just remove them; it won't meaningfully impact the PR size.